### ISSUE TITLE
MKL in conda, compilation error

### ISSUE THREAD

**Issue by bermanmaxim** [serge-sans-paille/pythran#726]
In my py2.7 conda environment that is linked with MKL, numpy_sys.get_info("blas") throws a distutils.errors.LinkError. I was able to use Pythran by changing the following lines around line.92 of config.py:
`
        numpy_blas = numpy_sys.get_info("blas")
`
to

```
        try:
            numpy_blas = numpy_sys.get_info("blas")
        except LinkError:
            numpy_blas = numpy_sys.get_info("mkl")
```

**Comment by serge-sans-paille** [comment]
can you tell me if the following works too?

```
for engine in ('mkl', 'atlas', 'blas'):
    try:
        numpy_blas = numpy_sys.get_info("blas")    
        ...
        break
    except LinkError:
        continue
```

**Comment by bermanmaxim** [comment]
I suppose you meant `get_info(engine)`. Yes that works, see pull request. Cheers

**Comment by serge-sans-paille** [comment]
Can you try #729 with setting

```
[compiler]
blas=mkl
```
in your ``~/.pythranrc`` ?

**Comment by serge-sans-paille** [comment]
Closed by #729
